usr/src/cmd/zfs/zfs_main.c

```diff
@@ -4875,6 +4886,9 @@ store_allow_perm(zfs_deleg_who_type_t type, boolean_t local, boolean_t descend,
 			ld[0] = ZFS_DELEG_LOCAL;
 		if (descend)
 			ld[1] = ZFS_DELEG_DESCENDENT;
+
+	default:
+		break;
```

Comment:
are there valid unhandled enum values here?  Is ignoring them the right thing to do?